resources/ec2/dhcp-config.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: ec2.services.k8s.aws/v1alpha1
```

Comment:
recommend naming the file `resources/ec2/v1alpha1/dhcpoptions.yaml` to ensure you name the file the same as the resource Kind.

Reply:
Thanks for the comment @jaypipes ! I've changed the filename to `dhcpoptions.yaml` in the original repository @RedbackThomson you can check it out.